model/Lite/SPDX-Lite.md

```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# SPDX-Lite
+
+## Summary
+
+Everything having to do with SPDX-Lite.
```

Comment:
Can you please elaborate here? We know this is related to SPDX-Lite because of the file name. A summary of the purpose and application of the profile would be more helpful.